fal/views/teams.py

```diff
@@ -0,0 +1,217 @@
+from dataclasses import dataclass
+import configparser
+
+from fal.clients.mfalncfm_main import session_scope
+from fal.models import Team, Season, TeamWeeklyAnime, Anime
+from fal.collect_series import get_season_from_database
+
+from typing import Dict, Tuple, List, Mapping, Sequence, TextIO, TYPE_CHECKING
+from sqlalchemy.orm import Session
+
+config = configparser.ConfigParser()
+config.read("config.ini")
+season_str: str = config["season info"]["season"]
+year: int = config.getint("season info", "year")
+
+
+@dataclass()
+class AnimeTeamCount:
+    num_teams: int
+    num_active: int
+
+
+# Implicitly combined strings for headcount intro and conclusion
+HEADCOUNT_INTRO_TEXT = (
+    "This is the headcount of the people who have been accepted in Fantasy "
+    "Anime League {} {}.\n\n"
+)
+
+HEADCOUNT_CONC_TEXT = (
+    "\nTotal teams: {}\n\nMake sure to check this list after "
+    "registering. We will be regularly updating it so you can see if your team "
+    "has been accepted or not.\n\nIf you're not on this list and the post's "
+    "last edit time is after you've sent your registration (give or take a few "
+    "hours), then you are [b]NOT[/b] registered. In that case, please contact "
+    "the FAL staff.\n\nAfter the registration is closed, we'll put up the team "
+    "list thread so you can check to make sure your team is correct."
+)
+
+TEAM_STATS_TEXT = (
+    "[u]Team Stats:[/u]\nThe list is ordered on how many times people chose a "
+    "certain anime in their team. The number in the brackets is the amount of "
+    "people who have the anime in their active team.\n\n"
+)
+
+SAME_SPLIT_TEXT = (
+    "[u]Same 7 series and same active team/bench distribution[/u]\nNumber of "
+    "unique teams: {}\n\nSame team chosen by:\n[list]"
+)
+
+SAME_NONSPLIT_TEXT = (
+    "[/list]\n[u]Same 7 series but different active team/bench "
+    "distribution[/u]\nNumber of unique teams: {}\n\nSame team chosen by:\n[list]"
+)
+
+SAME_ACTIVE_TEXT = (
+    "[/list]\n[u]Same active team (but can have different benched series)[/u]"
+    "\nNumber of unique active teams: {}\n\nSame team chosen by:\n[list]"
+)
+
+
+def get_team_from_season(season_str: str, year: int, session: Session) -> List[Team]:
+    season: Season = get_season_from_database(season_str, year, session)
+    return session.query(Team).filter(Team.season_id == season.id).all()
```

Comment:
get_season_from_database(...).teams will accomplish this.

Reply:
Oh I see, I forgot about the backpopulating.